lib/now-and-timers.cjs

```diff
@@ -1,5 +1,14 @@
 'use strict';
 const timers = require('timers');
 
-Object.assign(exports, timers);
+// Slightly higher than 24 days
+const MAX_INT32 = Math.pow(2, 31) - 1;
+
+const setSafeTimeout = (callback, ms) => {
+    const avaliableMs = ms < MAX_INT32 ? ms : MAX_INT32;
+
+    return timers.setTimeout(callback, avaliableMs)
+}
+
+Object.assign(exports, timers, { setSafeTimeout });
```

Comment:
It'd be better to assign `exports.setSafeTimeout` like we do for `exports.now` below.

Reply:
could you relaunch the ci? Probably it's occasional building error, cause check on node 18.0-windows. broken due to timeout.